dask_jobqueue/core.py

```diff
@@ -500,6 +504,17 @@ def __init__(
                 "jobqueue.%s.scheduler-options" % config_name, {}
             )
 
+        if protocol is None and security is not None:
+            protocol = "tls://"
+        if security is None and protocol is not None and protocol.startswith("tls"):
+            try:
+                security = Security.temporary()
+            except ImportError:
```

Comment:
I did not want to add an extra dependency. This is also the reason why tls should not be a default.

Reply:
Sounds good.